app/src/main/java/com/myskng/virtualchemlight/activity/UOActivity.kt

```diff
@@ -15,10 +15,9 @@ import android.view.*
 import com.myskng.virtualchemlight.R
 import com.myskng.virtualchemlight.uo.UOSensor
 import com.myskng.virtualchemlight.databinding.ActivityUoBinding
-import kotlinx.coroutines.experimental.Job
-import kotlinx.coroutines.experimental.android.UI
-import kotlinx.coroutines.experimental.launch
-import kotlin.coroutines.experimental.suspendCoroutine
+import kotlinx.coroutines.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/1cc3bbbb9f4a502ee80323c6980c854d59eccb44/app/src/main/java/com/myskng/virtualchemlight/activity/UOActivity.kt#L18"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>